### ISSUE TITLE
Can't use custom colors in EXE

### ISSUE THREAD

**Issue by Peter663217** [FlyTechVideos/BluescreenSimulator#186]
I tried to use a custom color in a exe file and it just gives me this error:
![image](https://github.com/FlyTechVideos/BluescreenSimulator/assets/84742317/a6de3a14-129a-4f69-aab6-aacbe00f25fc)
After clicking OK it displays the bluescreen, but with the default color.

**Comment by heavyutils** [comment]
That is an invalid color.

**Comment by Peter663217** [comment]
Yes but I created the exe file using the app, so it can't generate an invalid color, right?

**Comment by heavyutils** [comment]
Oh. Huh... try inputting you own color maybe? This is clearly a bug then

**Comment by Peter663217** [comment]
I even tried it using the command prompt but it also didn't work.
